src/Pages/Home/components/ListOfAlerts.tsx

```diff
@@ -1,61 +1,68 @@
+import { useTranslation } from "react-i18next"
 import { AlertSvg } from "../../../components/Icons/Alert";
 import { AlertIssueSvg } from "../../../components/Icons/AlertIssueSvg";
 import { WarnSvg } from "../../../components/Icons/WarnSvg";
 
+
```

Comment:
delete this line please

Reply:
From now we have eslint for issues like this.
in the Internal Network it's Done.